test/utils/test-helper.el

```diff
@@ -0,0 +1,50 @@
+;;; test-helper.el --- Clojure TS Mode: Non-interactive unit-test setup  -*- lexical-binding: t; -*-
+
+;; Copyright © 2022-2024 Danny Freeman
```

Comment:
Might be better to preserve the copyright attribution of the original code.

Reply:
Will add them back.